client/asset/btc/btc.go

```diff
@@ -1255,11 +1293,27 @@ func OpenSPVWallet(cfg *BTCCloneCFG, walletConstructor BTCWalletConstructor) (*E
 	}
 	btc.txHistoryDB.Store(txHistoryDB)
 
+	acctName := defaultAcctName
+	// Check if an account name was provided via wallet settings.
+	if configAcctName := cfg.WalletCFG.Settings[WalletAccountNameConfigKey]; configAcctName != "" {
+		acctName = configAcctName
+	}
+
+	var acctNumber uint32 = defaultAcctNum
+	// Check if an account number was provided via wallet settings.
+	if configAcctNumber := cfg.WalletCFG.Settings[WalletAccountNumberConfigKey]; configAcctNumber != "" {
+		accountNumber, err := strconv.ParseInt(configAcctNumber, 10, 64)
+		if err != nil {
+			return nil, fmt.Errorf("invalid config account number: %w", err)
+		}
+		acctNumber = uint32(accountNumber)
+	}
+
 	spvw := &spvWallet{
 		chainParams: cfg.ChainParams,
 		cfg:         walletCfg,
-		acctNum:     defaultAcctNum,
-		acctName:    defaultAcctName,
+		acctNum:     acctNumber,
+		acctName:    acctName,
```

Comment:
Let `spvw.wallet` define these. See how it is done similarly for dcr re `XCWalletAccounts`.